test/smoke_test/run_ac_smoke_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
```

Comment:
I'm not sure we need to split benchmark and accuracy smoke into two files. A person usially wants to run one something and get a result. It is already bad enough that we have units and integrations as two stages.

Reply:
discussed internally